apps/events/viewsets.py

```diff
@@ -135,6 +151,12 @@ def get_serializer_class(self):
                 return EmptySerializer
             case "delete_from_favorite":
                 return EmptySerializer
+            case "get_private_url":
```

Comment:
This endpoint no longer exists as I understand, we can remove it
And for all endpoints that use emptyserializer we can use case _: return EmptySerializer